src/FacadeAccountCreation/FacadeAccountCreation.API/Controllers/OrganisationsController.cs

```diff
@@ -100,6 +100,19 @@ public async Task<IActionResult> GetNationIdByOrganisationId(Guid organisationId
         }
     }
 
+    [HttpGet]
+    [Route("nation")]
+    [Consumes("application/json")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status204NoContent)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    public async Task<IActionResult> GetOrganisationNationByOrganisationExternalId(Guid organisationId)
```

Comment:
consider input validation so that you can return badrequest if invalid request

Reply:
I don't think we need to include the bad request attribute as I put this in error.

Reply:
done